src/main/java/edu/uw/cse403/Account.java

```diff
@@ -0,0 +1,22 @@
+package edu.uw.cse403;
+
+public class Account {
+    double principal,rate; int daysActive,accountType;
+    public static final int STANDARD = 0, BUDGET = 1, PREMIUM = 2, PREMIUM_PLUS = 3;
```

Comment:
I'm not familiar with the specification of the Account class, I see that PREMIUM and PREMIUM_PLUS are used interchangeably, could you provide more context on why we need these 4 types instead of just 2?

Reply:
Let's enum to represent account types since there's only finite possible cases here.